pkg/psmdb/backup/agent.go

```diff
@@ -62,7 +66,7 @@ func AgentContainer(cr *api.PerconaServerMongoDB, replsetName string) corev1.Con
 
 	if cr.CompareVersion("1.13.0") >= 0 {
 		c.Command = []string{psmdb.BinMountPath + "/pbm-entry.sh"}
-		c.Args = []string{"pbm-agent"}
+		c.Args = []string{"pbm-agent-entrypoint"}
```

Comment:
we need to use new arg only for >= `1.14.0` because we will not have it in old PBM image

Reply:
https://github.com/percona/percona-server-mongodb-operator/pull/1106/commits/4c8d3f094b9857ffe3f0560cc0562a01aa624224